siteimprove/admin/js/siteimprove.js

```diff
@@ -164,11 +183,14 @@
         }
 
         $(".recheck-button").click(function () {
+          $(this).attr("disabled", true);
```

Comment:
This is for another recheck button that is different from the Gutenberg one?

Reply:
We got a recheck button for Classic Editor & Gutenberg